### DISCUSSION TITLE
[Firemonkey] Options @exclude script from current domain in UI

### DISCUSSION THREAD

**Discussion by rmdhnreza** [erosman/support#532] [Ideas]
Is this feature can be added to Firemonkey?

![Fv2KoBk5Qg](https://user-images.githubusercontent.com/23427492/213204730-b18d8f93-4f15-4fc6-b0d6-32528fadf5b6.png)

Because, I need to edit script and add @exclude/@excludeMatches manually.

**Comment by erosman** [comment]
You can use **User Metadata** for that.
